### ISSUE TITLE
mIoU is too low

### ISSUE THREAD

**Issue by xinanbuyi** [Tramac/awesome-semantic-segmentation-pytorch#96]
1. The oscillation of loss function is severe

2. The segmentation effect is not good when using your own dataset, and the Miou is too low

Why, I look forward to your reply

**Comment by Tramac** [comment]
Hi, maybe the hyperparameters are not optimal or due to the reason of the dataset.

**Comment by swjtulinxi** [comment]
Hi，why is the miou lower than the paper，like pspnet，danet on cityscapes or voc2012，can we do a
same results as the paper?

**Comment by zzzzsh1** [comment]
i have the same problem

**Comment by XXMxxm220** [comment]
i also
